app/assets/javascripts/admin/order_cycles/controllers/order_cycles_controller.js.coffee

```diff
@@ -3,8 +3,12 @@ angular.module("admin.orderCycles").controller "OrderCyclesCtrl", ($scope, $q, C
   $scope.columns = Columns.columns
   $scope.saveAll = -> OrderCycles.saveChanges($scope.order_cycles_form)
   $scope.ordersCloseAtLimit = -31 # days
-  $scope.involvingFilter = 0
-  $scope.scheduleFilter = 0
+
+  $scope.resetSelectFilters = ->
+    $scope.scheduleFilter = 0
+    $scope.involvingFilter = 0
+    $scope.query = ''
+  $scope.resetSelectFilters()
```

Comment:
sorry @nikobozi, I'm not very angular-savy. When does this get executed? can you explain a bit?

Reply:
it's when the controller is loaded @sauloperez

Reply:
Great! thanks both!